### PR TITLE
New prop to prevent form submitting

### PR DESCRIPTION
**Pull request by kaskar2008**
## **Please check if the PR fulfills these requirements**
- [*] Tests for the changes have been added (for bug fixes / features)
- [*] Docs have been added / updated (for bug fixes / features)

## **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
This is a bug fix.

## **What is the current behavior?** (You can also link to an open issue here)
Issue #68 

## **What is the new behavior (if this is a feature change)?**
From now on you can prevent form submitting with a prop `prevent-submit` set to `true`.

## **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
Partly. This workaround includes a `form` html element into template.

### CODE REVIEW

**Comment by Raiondesu** [comment]
Maybe make this a default behaviour instead of another prop, which we already have tons of?

File: lib/vue-simple-suggest.vue [hunk 1]
```diff
@@ -3,48 +3,50 @@
...
-            :query="text">
-            <span>{{ displayProperty(suggestion) }}</span>
-          </slot>
-        </div>
-
-        <slot name="misc-item-below"
-          :suggestions="suggestions"
-          :query="text"
-        ></slot>
+    <form @submit="onSubmit">
```
[the remark this answers is not in the document]
Comment (kaskar2008) [reply lib/vue-simple-suggest.vue]: well, key events are differs from submittion events

**Comment by kaskar2008** [comment]
that's why this is just a patch. we need this prop ASAP, later we can think about it deeper.

File: lib/vue-simple-suggest.vue (hunk 1 above)
[the remark this answers is not in the document]
Comment (kaskar2008) [reply lib/vue-simple-suggest.vue]: Nvm. Got rid of that form anyway

**Comment by kaskar2008** [comment]
@Raiondesu No form for you

**Pull request merged**